test/3_givenProvider-ganache.js

```diff
@@ -30,7 +30,7 @@ describe('web.providers.givenProvider (ganache)', function(){
         assert.equal(1, web3.currentProvider.listenerCount('data'))
         assert.equal(1, web3.currentProvider.listenerCount('connect'))
         assert.equal(1, web3.currentProvider.listenerCount('error'))
-        assert.equal(1, web3.currentProvider.listenerCount('close'))
```

Comment:
can we add both the events to ensure we're backwards compatible?
Perhaps add a todo to remove `close` in the future

Reply:
this is just tests

Reply:
Yeah I can add the close event to the tests too.

Reply:
@GregTheGreek Done